oximeter/oximeter/schema/switch-table.toml

```diff
@@ -0,0 +1,106 @@
+format_version = 1
+
+[target]
+name = "switch_table"
+description = "A table on a Sidecar switch ASIC"
+authz_scope = "fleet"
+versions = [
+    { version = 1, fields = [ "rack_id", "sled_id", "sidecar_id", "table" ] },
+]
+
+[[metrics]]
+name = "capacity"
+description = "Maximum number of entries in the table"
```

Comment:
Is this something we want to keep a metric for? It will only really change when the P4 program changes.

Reply:
It was there originally, and I'm generally trying to make this phase a translation-only. That's not entirely true, though, so if you think this isn't worthwhile, we can just drop it.

Reply:
It should not be a blocker, and changing this would seem to require changes in Dendrite. Perhaps @Nieuwejaar has some insight on whether this is worth keeping.

Reply:
I will leave it for now then. We have the facility to drop individual timeseries now, so expunging it won't be difficult. Thanks for the review @rcgoodfellow!